apps/condo/pages/_app.tsx

```diff
@@ -82,11 +85,11 @@ const MenuItems: React.FC = () => {
                 />
                 <MenuItem
                     path={'/ticket'}
-                    icon={ThunderboltFilled}
+                    icon={ThunderboltFilledWithBadge}
```

Comment:
You are not passing count to the icon and it's not visible in collapsed state.

Reply:
According to design, it should not be visible

Reply:
It should. Take a look at https://www.figma.com/file/P827yyueCLRT6o4gyNtzY6/CRM.-Prod-(Ant)?node-id=9114%3A73492